sherpa-ncnn/csrc/sherpa-ncnn.cc

```diff
@@ -48,100 +45,40 @@ You can download pre-trained models from the following repository:
 
     return 0;
   }
```

Comment:
Could you update
https://github.com/k2-fsa/sherpa-ncnn/blob/master/.github/scripts/run-test.sh
to add tests for modified beam search?